spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -333,6 +332,16 @@ object StandardizationJob {
     standardizedDF.writeInfoFile(pathCfg.outputPath)
   }
 
+  private def handleControlInfoValidation(): Unit = {
+    ControlInfoValidation.addRawAndSourceRecordCountsToMetadata() match {
+      case Failure(ex: ValidationException) => conf.getString("control.info.validation") match {
+        case "strict" => throw ex
+        case "warning" => log.warn(ex.msg)
+        case _ =>
```

Comment:
This works as described, however, I wonder if it truly desired that invalid/misspelled value results in no action. I am personally a fan of strictly checking that only allowed enum-like values are used and verbosely failing otherwise.

Reply:
I was also thinking about this one. What do the others think? @benedeki @yruslan @Zejnilovic

Reply:
I'd prefer strict checking as well. Otherwise, a typo (i.e. `warming`) could lead to unexpected behavior that is not easy to spot.

Reply:
options are strict, warning and none? I would just check for `warning` and `none`, everything else is a `throw`.